mlir-sycl/lib/Conversion/SYCLToMath/SYCLToMath.cpp

```diff
@@ -33,11 +33,35 @@ namespace {
 template <typename SYCLOpT, typename MathOpT>
 struct OneToOneMappingPattern : public OpConversionPattern<SYCLOpT> {
   using OpConversionPattern<SYCLOpT>::OpConversionPattern;
+
   LogicalResult
   matchAndRewrite(SYCLOpT op, typename SYCLOpT::Adaptor adaptor,
```

Comment:
I agree with @victor-eds, having a separate pattern for non-MLIR-native types might make sense, especially given the name of this pattern (with wrap and unwrap, it's not really 1:1 anymore).

Reply:
Done.